lib/webdriveragent.js

```diff
@@ -111,15 +111,14 @@ class WebDriverAgent {
   createXcodeBuildSubProcess () {
 
     let args = [
-      '-workspace',
+      '-project',
       this.agentPath,
       '-scheme',
-      'XCTUITestRunner',
+      'WebDriverAgentRunner',
```

Comment:
@jlipps this is the new scheme in the webdriveragent. the submodule pointer for WebDriverAgent is pointing to a relatively outdated webdriveragent commit.

Im proposing a merge [here]( https://github.com/appium/WebDriverAgent/pull/2/files#diff-354f30a63fb0907d4ad57269548329e3R14) any reason why we dont want to use the latest? it works.